pkg/certificates/constants.go

```diff
@@ -33,3 +35,14 @@ const (
 	SecretCertKey   = "public-cert.pem"
 	SecretPKKey     = "private-key.pem"
 )
+
+func DataPlaneRoutingName(rountingId string) string {
+	if rountingId == "" {
+		rountingId = "0"
```

Comment:
Could we use something more intuitive eg. "default-id"?  Also there is a typo: s/rounting/routing.